### DISCUSSION TITLE
In Lesson 3, in the summary page you call the third problem ...Oversight..., but in the video later, you call it ...enforcement...

### DISCUSSION THREAD

**Discussion by gonzalo-munillag** [OpenMined/courses#142] [Feedback] [answered]
In Lesson 3, in the summary page you call the third problem ...Oversight..., but in the video later, you call it ...enforcement... It is the same but perhaps it is something you might care about.
Cheers! Loving the course.

**Comment by em-blue** [comment, marked as answer]
Hey @gonzalo-munillag! Both terms are correct :-) At one point, Andrew says "oversight, or enforcement" or something like that to show that they're both equivalent terms in this case. Glad you're paying attention though - enjoy the course!
